ios/Classes/NetIamyellowTiwsWSProxy.m

```diff
@@ -121,7 +124,14 @@ -(void)close:(id)args
 
 -(void)send:(id)msg
 {
-    ENSURE_SINGLE_ARG(msg, NSString);
+    ENSURE_SINGLE_ARG(msg, NSObject);
+    if ([msg isKindOfClass:[NSString class]]) {
+        msg = [TiUtils stringValue:msg];
+    } else if ([msg isKindOfClass:[TiBlob class]]) {
+        msg = [msg data];
+    } else {
+        NSAssert(NO, @"Argument type must be NSString or TiBlob");
+    }
```

Comment:
I just amended my original commit to add the braces and the assertion. Hopefully it's ok now.